.github/workflows/copilot-deploy-pipeline.yml

```diff
@@ -17,9 +17,20 @@ jobs:
   build:
     uses: ./.github/workflows/copilot-build-backend.yml
 
-  int:
-    needs: build
```

Comment:
I'm not sure it's a good idea to deploy the infra before we verify the code builds.